interpreter/cling/include/cling/Interpreter/Transaction.h

```diff
@@ -268,6 +273,16 @@ namespace cling {
       return const_reverse_nested_iterator(0);
     }
 
+    void addClangModule(clang::Module* M) {
+      if (!M) return;
+      if (std::find(m_cxxmodules.begin(), m_cxxmodules.end(), M) == m_cxxmodules.end())
```

Comment:
I suppose deserialization happens mostly local to a PCM? Then consider using `rbegin()`, `rend()` to look at the latest first!